src/Silex/Provider/ValidatorServiceProvider.php

```diff
@@ -46,7 +48,21 @@ public function register(Container $app)
         };
 
         $app['validator.mapping.class_metadata_factory'] = function ($app) {
-            return new LazyLoadingMetadataFactory(new StaticMethodLoader());
+            $loader = null;
```

Comment:
the `else` performance better than always creating an instance of the `StaticMethodLoader` and than not use it if annotation usage is configured